apps/src/blockly/addons/cdoScrollbar.ts

```diff
@@ -1,19 +1,25 @@
+import {ExtendedWorkspaceSvg} from '@cdo/apps/blockly/types';
+
 /**
  * Initializes the horizontal and vertical scrollbars in a workspace.
  *
  * @param {Object} workspace - The workspace object containing scrollbars and metrics.
- * @param {Blockly.ScrollbarPair} workspace.scrollbar - A set of scrollbars.
- * @param {Blockly.Scrollbar} workspace.scrollbar.hScroll - The horizontal scrollbar.
- * @param {Blockly.Scrollbar} workspace.scrollbar.vScroll - The vertical scrollbar.
  * @returns {void} This function does not return any value.
  */
-export function initializeScrollbarPair(workspace) {
+export function initializeScrollbarPair(workspace: ExtendedWorkspaceSvg) {
+  if (!workspace.scrollbar?.hScroll || !workspace.scrollbar?.vScroll) {
+    return;
+  }
   // In Core Blockly, pairs of scrollbars are always visible because the workspace
   // is always larger than the viewport. See the Blockly Playground for an example.
   // https://blockly-demo.appspot.com/static/tests/playground.html
   // Only non-paired scrollbars can have their visibility toggled.
-  workspace.scrollbar.hScroll.pair = false;
-  workspace.scrollbar.vScroll.pair = false;
+  // This is a hack, Blockly has pair set to be readonly. We force it to change here
+  // so we can unpair the two scrollbars.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (workspace.scrollbar.hScroll as any).pair = false;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (workspace.scrollbar.vScroll as any).pair = false;
```

Comment:
I'm not sure if it's helpful or not, but these should both be instances of the `Blockly.Scrollbar` class.

Reply:
Unfortunately `Blockly.scrollbar` does not allow resetting of `pair` so I had to cast it to hack around that.

Reply:
Ah, that tracks. Thanks for explaining.